src/App.tsx

```diff
@@ -1,14 +1,54 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
+enum Options {
+  active = 'active',
+  completed = 'completed',
+  all = 'all',
+}
```

Comment:
```suggestion
enum Options {
  Active = 'active',
  Completed = 'completed',
  All = 'all',
}
```
Enum keys should be capitalized